style.css

```diff
@@ -1,3 +1,3 @@
-h1{
-  color:blue;
```

Comment:
Testing the comments: remember to set a space between the selector and the curly brace!